lite/Dockerfile

```diff
@@ -44,17 +44,23 @@ RUN pip3 install ledgerblue
 
 ARG GIT_SERVER=https://github.com/LedgerHQ
 
+# Unified SDK
+ENV LEDGER_SECURE_SDK=/opt/ledger-secure-sdk
+RUN git clone "$GIT_SERVER/ledger-secure-sdk.git" "$LEDGER_SECURE_SDK"
+
 # Latest Nano S SDK
 ENV NANOS_SDK=/opt/nanos-secure-sdk
 RUN git clone --branch 2.1.0 --depth 1 "$GIT_SERVER/nanos-secure-sdk.git" "$NANOS_SDK"
 
-# Latest Nano X SDK
+# Latest Nano X SDK (OS nanox_2.1.0)
 ENV NANOX_SDK=/opt/nanox-secure-sdk
-RUN git clone --branch 2.0.2-2 --depth 1 "$GIT_SERVER/nanox-secure-sdk.git" "$NANOX_SDK"
+RUN git -C "$LEDGER_SECURE_SDK" worktree add "$NANOX_SDK" v1.1.0
```

Comment:
Typo on version number v1.1.0 should be v2.1.0 @xchapron-ledger

Reply:
niet, nanox_2.1.0 uses API_LEVEL_1, and therefore v1.1.0 SDK tag

Reply:
It's confusing then